### DISCUSSION TITLE
Cannot see the token after clicking generate token

### DISCUSSION THREAD

**Discussion by FlyingOnion** [community/community#51284] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

https://user-images.githubusercontent.com/17760761/228400637-c5257fba-1e37-4ae3-9485-dcac353156a5.mp4

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @FlyingOnion ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
